datafusion/optimizer/src/limit_push_down.rs

```diff
@@ -490,14 +497,14 @@ mod test {
 
         let plan = LogicalPlanBuilder::from(table_scan)
             .project(vec![col("a")])?
-            .limit(1000)?
-            .offset(10)?
+            .limit(None, Some(1000))?
+            .limit(Some(10), None)?
             .build()?;
 
-        let expected = "Offset: 10\
-        \n  Limit: 1000\
+        let expected = "Limit: skip=10, fetch=None\
```

Comment:
Perhaps a future optimization could consolidate these together into a single `Limit` -- I don't think it is particularly important, just to be clear